src/scripts/StoreRestApi.js

```diff
@@ -1,15 +1,20 @@
 import axios from 'axios';
 import Sitemap from './Sitemap';
 import StorePouchDB from './StorePouchDB';
+import urlJoin from 'url-join';
 
 export default class StoreRestApi {
-	constructor(config) {
+	constructor(config, baseUrl, sitemapsPath = 'sitemaps/') {
 		this.localDataStore = new StorePouchDB(config);
 		this.axiosInstance = axios.create({
-			baseURL: config.restUrl,
+			baseURL: baseUrl,
 		});
 		this.axiosInstance.defaults.headers.post['Content-Type'] = 'application/json';
 		this.axiosInstance.defaults.headers.put['Content-Type'] = 'application/json';
+		this.sitemapsPath = sitemapsPath;
+	}
+
+	postInit() {
```

Comment:
I don't like that this method is called at different time for talisman and rest. I suggest either
- rename it to what it actually does (sets up response interceptors)
- unify response interception so it works for both storages: for example, check content type only for sitemap responses 
- move content type check to response handlers in other methods explicitly